### ISSUE TITLE
[bluetooth.bluez] tinyb characteristics disappear

### ISSUE THREAD

**Issue by cpmeister** [openhab/openhab-addons#7010]
> 2020-02-14 08:49:39.955 [WARN ] [bluetooth.bluez.BlueZBluetoothDevice] - Characteristic '0000fe51-0000-1000-8000-00805f9b34fb' is missing on device '02:16:E9:71:63:8A'.
2020-02-14 08:49:40.978 [WARN ] [bluetooth.bluez.BlueZBluetoothDevice] - Characteristic '0000fe51-0000-1000-8000-00805f9b34fb' is missing on device '02:16:E9:71:63:8A'.
2020-02-14 08:49:41.005 [WARN ] [bluetooth.bluez.BlueZBluetoothDevice] - Characteristic '0000fe51-0000-1000-8000-00805f9b34fb' is missing on device '02:16:E9:71:63:8A'.
2020-02-14 08:50:38.884 [WARN ] [bluetooth.bluez.BlueZBluetoothDevice] - Characteristic '0000fe51-0000-1000-8000-00805f9b34fb' is missing on device '02:16:E9:71:63:8A'.

**Comment by cpmeister** [comment]
> Feb 14 09:20:24 openhab bluetoothd[18824]: ATT bt_io_connect(02:16:E9:71:63:8A): connect: Cannot allocate memory (12)
Feb 14 09:20:54 openhab bluetoothd[18824]: ATT bt_io_connect(02:16:E9:71:63:8A): connect: Cannot allocate memory (12)
Feb 14 09:21:24 openhab bluetoothd[18824]: ATT bt_io_connect(02:16:E9:71:63:8A): connect: Cannot allocate memory (12)

**Comment by J-N-K** [comment]
Is this a bug in tinyb or in OH?

**Comment by cpmeister** [comment]
@J-N-K I still don't know. Restarting my Raspi fixes the issue, but I don't know why it is running out of memory.
I have plenty of ram, so I thought it might be complaining about not enough jvm memory.
I upped the max memory for the jvm and it seemed to help the issue, although I haven't seen this issue in a while, I have still seen it occur even when jvm memory is not maxed out. 
I don't know if this is an issue that can be dealt with within the OH codebase or not, but I posted the issue here mostly for record keeping purposes.
